lib.php

```diff
@@ -1470,6 +1473,7 @@ function facetoface_download_attendees($facetofacename, $session, $attendees, $f
         }
         $worksheet->write_string($row, $column++, get_string('status_'.facetoface_get_status($attendee->statuscode), 'facetoface'),
                 ['border' => 1, 'v_align' => 'top']);
+        $worksheet->write_date($row, $column++, $attendee->timecreated, $format);
```

Comment:
Hey @djarran it looks like we will have to write the date as a string using the `write_string()` method to be able to format the date with `userdate()`